lib/js/src/compose/types/page-block/comment.ts

```diff
@@ -21,7 +22,8 @@ const defaults: Readonly<Options> = Object.freeze({
   sortDirection: '',
   referenceField: '',
   refreshRate: 0,
-  magnifyOption: ''
+  magnifyOption: '',
+  manualRefresh: false
```

Comment:
keep the refresh options one after the other, refreshEnabled, refreshRate. Just so you can see they are connected